changelogs/CHANGELOG-v0.3.0.md

```diff
@@ -0,0 +1,38 @@
+The v0.3.0 release is our second official minor release of the the bpfd project :tada
```

Comment:
Should we point to the changelog from the website? I didn't realize we had these for the previous releases until I saw this diff.

Reply:
hrm maybe in a follow up? For now they just get automatically put on the release page.

Reply:
Sorry, what's "the release page"?

Reply:
Nevermind, I just found it. In github the releases. I was looking in the website.